livsns/uploads/log/2015/08/13/configuare.php.txt

[2015-08-13 16:50:48]
/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 1198:
&nbsp;&nbsp;/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 969:
&nbsp;&nbsp;&nbsp;&nbsp;/Users/mac/Events/Project/livsns/lib/class/curl.class.php on line 258:http://vapi1.dev.hogesoft.com/configuare.php?format=json&a=settings&lpip=127.0.0.1&m2o_ckey=OjEN52E9LieIe9yx8mfDZEpDlUnxuya9&appid=55&appkey=GLtPX7N7ijwb83wupXuIrEl1YvIeBbm7&access_token=3aa2a36810f2ff98c44d2247baff6a9f

[2015-08-13 16:50:48]
/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 1198:
&nbsp;&nbsp;/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 969:
&nbsp;&nbsp;&nbsp;&nbsp;/Users/mac/Events/Project/livsns/lib/class/curl.class.php on line 259:array (
  'a' => 'settings',
  'lpip' => '127.0.0.1',
  'm2o_ckey' => 'OjEN52E9LieIe9yx8mfDZEpDlUnxuya9',
  'appid' => '55',
  'appkey' => 'GLtPX7N7ijwb83wupXuIrEl1YvIeBbm7',
  'access_token' => '3aa2a36810f2ff98c44d2247baff6a9f',
)

[2015-08-13 16:50:48]
/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 1198:
&nbsp;&nbsp;/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 969:
&nbsp;&nbsp;&nbsp;&nbsp;/Users/mac/Events/Project/livsns/lib/class/curl.class.php on line 327:{"db":{"host":"db.dev.hogesoft.com","user":"root","database":"dev_media","charset":"utf8","pconnect":"","dbprefix":"liv_"},"api_dir":"\/web\/mediaserver4\/api\/mediaserver","define":{"DB_PREFIX":"liv_","APP_UNIQUEID":"mediaserver","UPLOAD_DIR":"\/web\/mediauploads\/media\/","TARGET_DIR":"\/web\/mediauploads\/mp4\/","FTP_UPLOAD_DIR":"\/web\/mediauploads\/","PICK_UP_DIR":"\/web\/mediauploads\/","MEDIAINFO_CMD":"\/usr\/local\/bin\/mediainfo","FFMPEG_CMD":"\/usr\/bin\/ffmpeg","MP4SPLIT_CMD":"\/usr\/local\/bin\/mp4split  -o ","FFMPED2TS_CMD":"\/usr\/local\/ffmpeg2.0.1\/bin\/ffmpeg","TS_DURATION":"10","APPID":"55","APPKEY":"GLtPX7N7ijwb83wupXuIrEl1YvIeBbm7","SOURCE_VIDEO_DOMIAN":"http:\/\/10.0.1.58\/mediauploads\/media\/","TARGET_VIDEO_DOMAIN":"vfile1.dev.hogesoft.com:233","WATER_POS":"0,0","MORE_BITRATE_SERVER":"61","MANDATORY_SERVER":"61","NOT_CREATE_ISMV":"0","SNAP_PIC_POS":"3","PUBLISH_SET_ID":"25","IS_TRANSCODE_URL":"1","MAX_CLOUD_DELETE":"10","MAX_CLOUD_DELETE_TIME":"36","INITED_APP":"true"},"base":{"is_open_xs":1,"token_expired":1800,"weight_search":["0","1","2","3","10","20","30","40","50","60","70","80","90"],"status_color":{"0":"#8ea8c8","1":"#17b202","2":"#f8a6a6","\u5f85\u5ba1\u6838":"#8ea8c8","\u5df2\u5ba1\u6838":"#17b202","\u5df2\u6253\u56de":"#f8a6a6","\u88ab\u6253\u56de":"#f8a6a6","\u5f55\u5236\u4e2d\u2026":"#f8a6a6","\u7b49\u5f85\u5f55\u5236":"#8ea8c8","\u5f55\u5236\u6210\u529f":"#17b202"},"state_search":["\u5168\u90e8\u72b6\u6001","\u672a\u5ba1\u6838","\u5df2\u5ba1\u6838","\u5df2\u6253\u56de"],"status_show":["\u672a\u5ba1\u6838","\u5df2\u5ba1\u6838","\u5df2\u6253\u56de"],"date_search":{"1":"\u6240\u6709\u65f6\u95f4\u6bb5","2":"\u6628\u5929","3":"\u4eca\u5929","4":"\u6700\u8fd13\u5929","5":"\u6700\u8fd17\u5929","other":"\u81ea\u5b9a\u4e49\u65f6\u95f4"},"App_card":{"host":"localhost","dir":"livsns\/api\/card\/"},"App_catalog":{"host":"localhost","dir":"livsns\/api\/catalog\/"},"App_constellation":{"host":"localhost","dir":"livsns\/api\/constellation\/"},"App_road":{"host":"localhost","dir":"livsns\/api\/road\/"},"App_tv_interact":{"host":"localhost","dir":"livsns\/api\/tv_interact\/"},"App_recycle":{"host":"10.0.1.40","dir":"livsns\/api\/recycle\/"},"App_material":{"host":"10.0.1.40","dir":"livsns\/api\/material\/"},"App_magazine":{"host":"10.0.1.40","dir":"livsns\/api\/magazine\/"},"App_cheapbuy":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/cheapbuy\/"},"App_logs":{"host":"10.0.1.40","dir":"livsns\/api\/logs\/"},"App_news":{"host":"10.0.1.40","dir":"livsns\/api\/news\/"},"App_tuji":{"host":"10.0.1.40","dir":"livsns\/api\/tuji\/"},"App_old_live":{"protocol":"http:\/\/","host":"api.dev.hogesoft.com","dir":"old_live\/"},"App_live":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/live_takeover\/"},"App_live_control":{"protocol":"http:\/\/","host":"api.dev.hogesoft.com","dir":"live_control\/"},"App_schedule":{"protocol":"http:\/\/","host":"api.dev.hogesoft.com","dir":"schedule\/"},"App_program":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/program\/"},"App_program_record":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/program_record\/"},"App_live_interactive":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/live_interactive\/"},"App_livmedia":{"host":"10.0.1.40","dir":"livsns\/api\/livmedia\/"},"App_servermonitor":{"host":"localhost","dir":"livsns\/api\/servermonitor\/"},"App_player":{"protocol":"http:\/\/","host":"api.dev.hogesoft.com","dir":"player\/","demovideo":"latest","demochannel":"latest"},"App_mediaserver":{"protocol":"http:\/\/","host":"vapi1.dev.hogesoft.com","dir":"","token":"aldkj12321aasd","port":"80","filename":"video.php"},"App_auth":{"host":"10.0.1.40","dir":"livsns\/api\/auth\/"},"App_statistics":{"host":"10.0.1.40","dir":"livsns\/api\/statistics\/"},"App_access":{"host":"10.0.1.40","dir":"livsns\/api\/access\/"},"App_share":{"host":"10.0.1.40","dir":"livsns\/api\/share\/"},"App_workbench":{"host":"10.0.1.40","dir":"livsns\/api\/workbench\/"},"App_publishcontent":{"host":"10.0.1.40","dir":"livsns\/api\/publishcontent\/"},"App_publishplan":{"host":"10.0.1.40","dir":"livsns\/api\/publishplan\/"},"App_publishconfig":{"host":"10.0.1.40","dir":"livsns\/api\/publishcontent\/"},"App_publishsys":{"host":"10.0.1.40","dir":"livsns\/api\/publishsys\/"},"App_block":{"host":"10.0.1.40","dir":"livsns\/api\/block\/"},"App_special":{"host":"10.0.1.40","dir":"livsns\/api\/special\/"},"App_travel":{"host":"10.0.1.40","dir":"livsns\/api\/travel\/"},"App_login":{"host":"10.0.1.40","dir":"livsns\/api\/member\/"},"App_members":{"host":"localhost","dir":"livsns\/api\/members\/"},"App_opinion":{"host":"10.0.1.40","dir":"livsns\/api\/opinion\/"},"App_livcms":{"host":"10.0.1.40","dir":"livsns\/api\/livcms\/"},"App_mk_publish_content":{"host":"10.0.1.40","dir":"hoge_2012\/cp\/mk\/"},"App_mkpublish":{"host":"localhost","dir":"livsns\/api\/mkpublish\/"},"App_mark":{"host":"10.0.1.40","dir":"livsns\/api\/mark\/"},"App_member":{"host":"10.0.1.40","dir":"livsns\/api\/member\/"},"App_cp_status":{"host":"10.0.1.40","dir":"livsns\/admin_api\/cp_status\/"},"App_status":{"host":"10.0.1.40","dir":"livsns\/api\/statuses\/"},"App_queue":{"host":"10.0.1.40","dir":"livsns\/api\/service\/"},"App_memcache":{"host":"10.0.1.40","dir":"livsns\/api\/service\/"},"App_memcached":{"host":"10.0.1.40","dir":"livsns\/api\/memcache\/"},"App_shorturl":{"host":"10.0.1.40","dir":"livsns\/api\/shorturl\/"},"App_cp_albums":{"host":"10.0.1.40","dir":"livsns\/api\/cp_albums\/"},"App_group":{"host":"10.0.1.40","dir":"livsns\/api\/group\/"},"App_notify":{"host":"10.0.1.40","dir":"livsns\/api\/notify\/"},"App_option":{"host":"10.0.1.40","dir":"livsns\/api\/action\/"},"App_team":{"host":"10.0.1.40","dir":"livsns\/api\/team\/"},"App_activity":{"host":"10.0.1.40","dir":"livsns\/api\/activity\/"},"App_appstore":{"host":"10.0.1.40","dir":"livsns\/api\/appstore\/"},"App_adv":{"host":"api.dev.hogesoft.com","dir":"adv\/"},"App_vote":{"host":"10.0.1.40","dir":"livsns\/api\/vote\/"},"App_message":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/message\/","token":"aldkj12321aasd"},"App_mobile":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/mobile\/"},"App_banword":{"host":"localhost","dir":"livsns\/api\/banword\/"},"App_message_received":{"host":"10.0.1.40","dir":"livsns\/api\/message_received\/"},"App_settings":{"host":"10.0.1.40","dir":"livsns\/api\/settings\/"},"App_photoedit":{"host":"10.0.1.40","dir":"livsns\/api\/photoedit\/"},"App_webapp":{"host":"localhost","dir":"livsns\/api\/webapp\/"},"App_payments":{"host":"localhost","dir":"livsns\/api\/payments\/"},"App_verifycode":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/verifycode\/"},"App_video_split":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/video_split\/"},"App_video_fast_edit":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/video_fast_edit\/"},"App_qrcode":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/qrcode\/"},"App_screen3syn":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/screen3syn\/"},"App_contribute":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/contribute\/"},"App_ticket":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/ticket\/"},"App_textsearch":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/textsearch\/"},"App_archive":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/archive\/"},"App_weather":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/weather\/"},"App_carpark":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/carpark\/"},"App_live_time_shift":{"protocol":"http:\/\/","host":"api.dev.hogesoft.com","dir":"live_time_shift\/"},"App_seekhelp":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/seekhelp\/"},"App_video_point":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/video_point\/"},"App_tv_play":{"protocol":"http:\/\/","host":"api.dev.hogesoft.com","dir":"tv_play\/"},"App_albums_app":{"host":"localhost","dir":"livsns\/api\/albums_app\/"},"App_dingdoneuser":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/dingdoneuser\/"},"App_app_plant":{"host":"localhost","dir":"livsns\/api\/app_plant\/"},"App_supermarket":{"host":"api.dev.hogesoft.com","dir":"supermarket\/"},"App_epaper":{"host":"localhost","dir":"livsns\/api\/epaper\/"},"App_cdn":{"host":"localhost","dir":"livsns\/api\/cdn\/"},"App_lbs":{"host":"localhost","dir":"livsns\/api\/lbs\/"},"App_gather":{"host":"localhost","dir":"livsns\/api\/gather\/"},"App_mood":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/mood\/"},"App_grade":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/grade\/"},"App_survey":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/survey\/"},"App_company":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/company\/"},"App_meeting":{"protocol":"http:\/\/","host":"10.0.1.40","dir":"livsns\/api\/meeting\/"},"App_searchtag":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/searchtag\/"},"App_feedback":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/feedback\/"},"App_jf_mall":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/jf_mall\/"},"App_dingdonestatistics":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/dingdonestatistics\/"},"App_gatherapi":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/gatherapi\/"},"App_im":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/im\/"},"App_userSpace":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/userSpace\/"},"App_live_interactive_new":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/live_interactive_new\/"},"App_qcon":{"protocol":"http:\/\/","host":"localhost","dir":"livsns\/api\/qcon\/"},"video_cloud":"","video_cloud_delete":2,"video_cloud_title":"\u4e50\u89c6\u4e91","video_file_cloud":2,"cloud_user":"","cloud_secret_key":"","video_type":{"allow_type":".wmv,.avi,.dat,.asf,.rm,.rmvb,.ram,.mpg,.mpeg,.3gp,.mov,.mp4,.m4v,.dvix,.dv,.dat,.mkv,.flv,.vob,.ram,.qt,.divx,.cpk,.fli,.flc,.mod,.m4a,.f4v,.3ga,.caf,.mp3,.vob,.aac,.amr,.ts"},"metadata":{"album":"","artist":"","comment":"","composer":"","copyright":"","creation_time":"","genre":"","title":""},"used_search_condition":[]},"dir_index":"0","debuged":0,"dbconnected":1,"connect_time":"0.000006"}

[2015-08-13 17:00:44]
/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 1198:
&nbsp;&nbsp;/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 969:
&nbsp;&nbsp;&nbsp;&nbsp;/Users/mac/Events/Project/livsns/lib/class/curl.class.php on line 258:http://vapi1.dev.hogesoft.com/configuare.php?format=json&a=settings&lpip=127.0.0.1&m2o_ckey=OjEN52E9LieIe9yx8mfDZEpDlUnxuya9&appid=55&appkey=GLtPX7N7ijwb83wupXuIrEl1YvIeBbm7&access_token=3aa2a36810f2ff98c44d2247baff6a9f

[2015-08-13 17:00:44]
/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 1198:
&nbsp;&nbsp;/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 969:
&nbsp;&nbsp;&nbsp;&nbsp;/Users/mac/Events/Project/livsns/lib/class/curl.class.php on line 259:array (
  'a' => 'settings',
  'lpip' => '127.0.0.1',
  'm2o_ckey' => 'OjEN52E9LieIe9yx8mfDZEpDlUnxuya9',
  'appid' => '55',
  'appkey' => 'GLtPX7N7ijwb83wupXuIrEl1YvIeBbm7',
  'access_token' => '3aa2a36810f2ff98c44d2247baff6a9f',
)

[2015-08-13 17:00:44]
/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 1198:
&nbsp;&nbsp;/Users/mac/Events/Project/livsns/api/livmedia/admin/vod.php on line 969:

livworkbench/conf/config.php
<?php
$gDBconfig = array(
	'host'     => 'db.dev.hogesoft.com', //221.226.87.26
	'user'     => 'root',
	'pass'     => 'hogesoft',
	'database' => 'dev_workbench',
	'charset'  => 'utf8',
	'pconncet' => 0,
);
define('DB_PREFIX','liv_');//定义数据库表前缀

define('DEBUG_MODE', false); //debug模式开关
define('DEVELOP_MODE',true); //开发模式开关

//测试客户安装配置
define('CUSTOM_APPID',29);
define('CUSTOM_APPKEY','OjEN52E9LieIe9yx8mfDZEpDlUnxuya9');

define('APPID', '55');
define('APPKEY', 'GLtPX7N7ijwb83wupXuIrEl1YvIeBbm7'); 


//define('GOOGLE_MAP_KEY', 'ABQIAAAAcR9WO5MhlzqSal8bHphuDRRyk9R03h4R4ZJDKZMqZnC8yqgEVRT2bGP3fqEsz7VY-3dTDzqLS7VQ7g');
define('GOOGLE_MAP_KEY', 'AIzaSyAdCOYn-vDh2HtKgkI5-w1v7hW7sa5XyDM');

define('DEFAULT_LOCATION', '31.998177x118.775253');

$gGlobalConfig['officeconvert']=array(
	'host' => 'officeconvert.hogesoft.com:8080',
	'dir' => 'converter/'
);
$gGlobalConfig['App_auth'] = array(
	'host' => 'api.dev.hogesoft.com',
	'dir' => 'auth/',
	'admin'=>'auth/admin/'
);

$gGlobalConfig['App_liv_mms']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/liv_mms/',
	'token' => '3e67f243c4965f5f233d2003cacfb16d',
);

$gGlobalConfig['App_appstore']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/appstore/',
	'token' => '3e67f243c4965f5f233d2003cacfb16d',
);

$gGlobalConfig['App_auth']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/auth/',
);
$gGlobalConfig['App_upserver']=array(
	'host' => 'vapi.dev.hogesoft.com',
	'dir' => 'api/',
	'token' => 'aldkj12321aasd',
);
$gGlobalConfig['App_mediaserver']=array(
	'host' => 'media.dev.hogesoft.com',
	'dir' => 'api/',
	'token' => 'aldkj12321aasd',
);
$gGlobalConfig['App_live']=array(
	'host' => 'api.dev.hogesoft.com',
	'dir' => 'live/',
);
$gGlobalConfig['App_livcms']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/livcms/',
);

$gGlobalConfig['App_ad']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/adv/',
);
$gGlobalConfig['App_weather']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/weather/',
);
$gGlobalConfig['App_publishconfig']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/publishcontent/',
);

$gGlobalConfig['App_material'] = array(
	'host' => '10.0.1.40',
	'dir' => 'livsns/api/material/',
	'token' => '3e67f243c4965f5f233d2003cacfb16d',
);

$gGlobalConfig['App_publishcontent']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/publishcontent/',
);
$gGlobalConfig['App_mcp_statues']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/mcp_statues/',
);
$gGlobalConfig['App_news']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/news/',
	
);
	
$gGlobalConfig['App_tuji']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/tuji/',
	
);
$gGlobalConfig['App_livmedia']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/livmedia/',
	
);
$gGlobalConfig['App_photoedit']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/photoedit/',
	'token' => '3e67f243c4965f5f233d2003cacfb16d',
);
//直播互动
$gGlobalConfig['App_interactive'] = array(
	'host' => 'dev.hogesoft.com',
	'mid' => array(
		'16' => 323,
		'13'  => 308,
	),
);
//专题配置
$gGlobalConfig['App_special']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/special/',
);
$gGlobalConfig['App_block']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/block/',
);
$gGlobalConfig['App_settings']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/settings/',
	'token' => '3e67f243c4965f5f233d2003cacfb16d',
);
$gGlobalConfig['App_publishsys']=array(
	'host' => 'localhost',
	'dir' => 'livsns/api/publishsys/',	
);
$gGlobalConfig['App_catalog'] =array(
	'host' => 'localhost',
	'dir' => 'livsns/api/catalog/',
	
);
$gGlobalConfig['App_textsearch'] = array(
	'protocol' => 'http://',
	'host' => 'localhost',
	'dir' => 'livsns/api/textsearch/',
);
$gGlobalConfig['App_verifycode'] = array(
    'protocol' => 'http://',
    'host' => 'localhost',
    'dir' => 'livsns/api/verifycode/',
);

$gGlobalConfig['App_livmedia'] = array(
    'protocol' => 'http://',
        'host' => 'localhost',
            'dir' => 'livsns/api/livmedia/',
);

$gGlobalConfig['version'] = '5.0.0';

define('INITED_APP', true);
?>
